client/my-sites/stats/activity-log-upgrade-notice/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import QuerySitePlans from 'components/data/query-site-plans';
+import Banner from 'components/banner';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { isCurrentPlanPaid } from 'state/sites/selectors';
+
+class ActivityLogUpgradeNotice extends Component {
+	static propTypes = {
+		siteId: PropTypes.number.isRequired,
+
+		// Connected props
+		siteSlug: PropTypes.string.isRequired,
+		hasPlan: PropTypes.bool.isRequired,
+
+		// localize
+		translate: PropTypes.func.isRequired,
+	};
+
+	render() {
+		if ( null === this.props.hasPlan || this.props.hasPlan ) {
+			return false;
+		}
+
+		const {
+			siteId,
+			siteSlug,
+			translate,
+		} = this.props;
+
+		return [
+			<QuerySitePlans siteId={ siteId } />,
+			<Banner
+				plan="personal-bundle"
+				href={ `/plans/${ siteSlug }` }
+				title={ translate( 'Upgrade your Jetpack to rewind to specific events.' ) }
```

Comment:
we might want some copy review on this string

Reply:
CC @nicoleckohler @michelleweber

Reply:
Would suggest "Upgrade your Jetpack plan to restore your site to events in the past." @dmsnell